Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -442,7 +443,8 @@ void Wiimote::Update()
   // We duplicate data reports to accomplish this.
   // Unfortunately this breaks detection of motion gestures in some games.
   // e.g. Sonic and the Secret Rings, Jett Rocket
```

Comment:
It would probably be good to mention in the comment that there are things other than speaker data which break if the input rate isn't around 200 Hz.

Reply:
Done.